molecule/provisioner/ansible.py

```diff
@@ -930,6 +930,10 @@ def _get_modules_directories(self):
                 "/usr/share/ansible/plugins/modules",
             ]
         )
+
+        if os.environ.get("ANSIBLE_LIBRARY"):
+            paths.extend([util.abs_path(os.environ.get("ANSIBLE_LIBRARY"))])
```

Comment:
I think this should have been added first because ANSIBLE_LIBRARY being passed in likely should take priority over the computed one. In our instance the system path takes precedence over the testing one if the same module is in both places.

ANSIBLE_LIBRARY gets set to:
-> /home/stack/test-python/lib/python3.6/site-packages/molecule/provisioner/ansible/plugins/modules
-> /home/stack/.cache/molecule/tripleo_network_config/default/library
-> /home/stack/tripleo-ansible/tripleo_ansible/roles/tripleo_network_config/library
-> /home/stack/.ansible/plugins/modules
-> /usr/share/ansible/plugins/modules
-> /home/stack/tripleo-ansible/tripleo_ansible/roles.galaxy/config_template/library
-> /home/stack/tripleo-ansible/tripleo_ansible/ansible_plugins/modules
-> /home/stack/.cache/ansible-lint/128d82/modules
-> /home/stack/tripleo-ansible/tripleo_ansible/ansible_plugins/modules

meaning the same module in /usr/share/ansible/plugins/mopdules takes precedence over out testing version in /home/stack/tripleo-ansible/tripleo_ansible/ansible_plugins/modules